src/quacc/runners/prep.py

```diff
@@ -57,10 +59,16 @@ def calc_setup(
     )
 
     # Create a tmpdir for the calculation within the scratch_dir
-    time_now = datetime.now(timezone.utc).strftime("%Y-%m-%d-%H-%M-%S-%f")
-    tmpdir = Path(
-        mkdtemp(prefix=f"quacc-tmp-{time_now}-", dir=SETTINGS.SCRATCH_DIR)
-    ).resolve()
+    # Just a test to see if we can avoid heavy io
+    if tmpdir:
+        prefix = f"quacc-tmp-common-{tmpdir}"
+        tmpdir = Path(SETTINGS.SCRATCH_DIR / prefix)
+        tmpdir.mkdir(parents=True, exist_ok=True)
```

Comment:
I remain hesitant about changes to the path handling for the reasons previously described. You don't want files to live in scratch after a calculation has been completed because the scratch directory may often be on a filesystem that is purged, and there is no guarantee for when the follow-up job in the workflow will be executed. The scratch directory is not meant to persistent. I'd rather have file transfers than scenarios subject to data loss.

If the problem is that the copy operations are too expensive, then the user should define the `RESULTS_DIR` to be on the same filesystem as `SCRATCH_DIR`, since then it should be a fairly trivial operation.